pkgs/servers/mlflow-server/default.nix

```diff
@@ -1,36 +1,43 @@
-{ python3, writeText}:
+{
+  python3Packages,
+  writeText,
+  python,
```

Comment:
The top-level `python` attribute is a deprecated alias, and that's detected by ofBorg eval.  Perhaps just use `python3Packages.python`?  (for easier overrides)

Reply:
Or you could remain with the previous way of just passing `python3` and using `python3.pkgs`.  (I don't know what's the best practice here)